svthalia/concrexit
# Add withdrawal date to batches

Closes #1254 

### Summary
A withdrawal date is added to batches

### How to test
1. Create a batch, see the 'withdrawal date' being suggested 14 days from now
2. Process the batch, the email should contain the withdrawal date
3. Check out the payment list view, it should contain a settlement date
